### DISCUSSION TITLE
12-13

### DISCUSSION THREAD

**Discussion by miniposi** [ChristmasTest/christmas#84] [Daily_Meeting]
### 🎅🏻 회의 내용

**어제 한 내용 공유**
- 수영님 : 결과 페이지 style
- 수님 : 결과 페이지 디자인
- 슬희님 : 메인 페이지 스타일 수정 / 모달 컴포넌트
- 윤혁님 : storybook 에러 잡음
<br />

### ❄️ 오늘 할일
* 코딩하기
* 운동하기
* 🚨 **벨 boolean 값 바꾸기**
* 🚨 결과 페이지 배경화면 바꾸기
* 🚨 로딩 만들기
<br />

### ⛄️ TMI
오늘은 없나봐요

**Comment by hanseulhee** [comment]
수님 깃헙 아이디 지금보니까 미니 포시엿네 !!

**Reply by miniposi** [reply]
흠 ... 이걸 이제야? 🤔

**Reply by hanseulhee** [reply]
제 아이디는 뜻 몬데요
